app/src/main/java/io/github/droidkaigi/confsched/fragment/SettingsFragment.java

```diff
@@ -55,16 +58,26 @@ public void onAttach(Context context) {
     private void initView() {
         binding.txtLanguage.setText(LocaleUtil.getCurrentLanguage(getActivity()));
         binding.languageSettingsContainer.setOnClickListener(v -> showLanguagesDialog());
-        binding.notificationSwitchRow.init(PrefUtil.KEY_NOTIFICATION_SETTING, true);
-        binding.localTimeSwitchRow.init(PrefUtil.KEY_SHOW_LOCAL_TIME, false);
+
+        boolean shouldNotify = DefaultPrefsSchema.get(getContext()).getNotificationFlag(true);
```

Comment:
I think we don't have to use application context. Generally speaking, memory leak could occur when given context is used in different lifetime. Meanwhile in kvs-schema, `DefaultPrefsSchema.get` runs synchronoushly and this library doesn't hold a reference of context.